### PR TITLE
add(epp): hardware-sensors-rittal-cmc3-snmp

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [X] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Contenu du pack
+
+### Modèles
+
+Le connecteur de supervision **Rittal CMC3 SNMP** apporte un modèle d'hôte :
+
+* **HW-Sensor-Rittal-Cmc3-SNMP**
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md:14]:
```suggestion
* **HW-Sensor-Rittal-Cmc3-SNMP-custom**
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="HW-Sensor-Rittal-Cmc3-SNMP" label="HW-Sensor-Rittal-Cmc3-SNMP">
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md:20]:
```suggestion
<TabItem value="HW-Sensor-Rittal-Cmc3-SNMP-custom" label="HW-Sensor-Rittal-Cmc3-SNMP-custom">
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md [hunk 3]
```diff
@@ -0,0 +1,453 @@
...
+
+| Alias       | Modèle de service                       | Description                                               |
+|:------------|:----------------------------------------|:----------------------------------------------------------|
+| Load        | HW-Sensors-Rittal-Cmc3-Load-SNMP        | Contrôle la charge système                                |
+| Unit-Status | HW-Sensors-Rittal-Cmc3-Unit-Status-SNMP | Contrôle le statut de l'unité                             |
+| Uptime      | HW-Sensors-Rittal-Cmc3-Uptime-SNMP      | Durée depuis laquelle le serveur tourne sans interruption |
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **HW-Sensor-Rittal-Cmc3-SNMP** est utilisé.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md:28]:
```suggestion
> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **HW-Sensor-Rittal-Cmc3-SNMP-custom** est utilisé.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md [hunk 4]
```diff
@@ -0,0 +1,453 @@
...
+| CRITICALSTATUS | Define the conditions to match for the status to be CRITICAL (default: '%{status} =~ /notAvail\|lost\|error/i'). You can use the following variables: %{status}                 | %{status} =~ /notAvail|lost|error/i                 |             |
+| EXTRAOPTIONS   | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                                             | --verbose                                           |             |
+
+</TabItem>
+<TabItem value="Load" label="Load">
+
+| Macro               | Description                                                                                         | Valeur par défaut | Obligatoire |
+|:--------------------|:----------------------------------------------------------------------------------------------------|:------------------|:------------|
+| WARNINGUNITLOAD10M  |                                                                                                     |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md:228]: missing descriptions

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md [hunk 5]
```diff
@@ -0,0 +1,453 @@
...
+| system.uptime.seconds | s     |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+### Configuration SNMP
+
+Afin de superviser votre équipement, le SNMP v2 ou v3 doit être configuré.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md:96]:
```suggestion
Afin de superviser votre ressource en SNMP,  il est nécessaire de configurer l'agent SNMP v2 ou v3
sur la ressource comme indiqué sur la documentation officielle du constructeur.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md [hunk 6]
```diff
@@ -0,0 +1,453 @@
...
+| --snmp-tls-trust-cert                      | A trusted CA certificate used to verify a remote host's certificate. If you use this option, you must also define --snmp-tls-their-hostname.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | SNMP   |
+
+#### Options des modes
+
+Les options spécifiques aux modes sont listées ci-dessus :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md:387]:
```suggestion
Les options spécifiques aux modes sont listées ci-dessous :
```

File: pp/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md [hunk 7]
```diff
@@ -0,0 +1,452 @@
...
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Pack assets
+
+### Templates
+
+The Monitoring Connector **Rittal CMC3 SNMP** brings a host template:
+
+* **HW-Sensor-Rittal-Cmc3-SNMP**
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md:14]:
```suggestion
* **HW-Sensor-Rittal-Cmc3-SNMP-custom**
```

File: pp/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md [hunk 8]
```diff
@@ -0,0 +1,452 @@
...
+<Tabs groupId="sync">
+<TabItem value="HW-Sensor-Rittal-Cmc3-SNMP" label="HW-Sensor-Rittal-Cmc3-SNMP">
+
+| Service Alias | Service Template                        | Service Description                                  |
+|:--------------|:----------------------------------------|:-----------------------------------------------------|
+| Load          | HW-Sensors-Rittal-Cmc3-Load-SNMP        | Check system load                                    |
+| Unit-Status   | HW-Sensors-Rittal-Cmc3-Unit-Status-SNMP | Check unit status                                    |
+| Uptime        | HW-Sensors-Rittal-Cmc3-Uptime-SNMP      | Time since the server has been working and available |
+
+> The services listed above are created automatically when the **HW-Sensor-Rittal-Cmc3-SNMP** host template is used.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md:27]:
```suggestion
> The services listed above are created automatically when the **HW-Sensor-Rittal-Cmc3-SNMP-custom** host template is used.
```

File: pp/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md [hunk 9]
```diff
@@ -0,0 +1,452 @@
...
+
+</TabItem>
+<TabItem value="Not attached to a host template" label="Not attached to a host template">
+
+| Service Alias | Service Template                    | Service Description    | Discovery  |
+|:--------------|:------------------------------------|:-----------------------|:-----------|
+| Devices       | HW-Sensors-Rittal-Cmc3-Devices-SNMP | Check attached devices | X          |
+
+> The services listed above are not created automatically when a host template is applied. To use them, [create a service manually](/docs/monitoring/basic-objects/services), then appy the service template you want.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md:36]:
```suggestion
> The services listed above are not created automatically when a host template is applied. To use them, [create a service manually](/docs/monitoring/basic-objects/services), then apply the service template you want.
```

File: pp/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md [hunk 10]
```diff
@@ -0,0 +1,452 @@
...
+| system.uptime.seconds | s     |
+
+</TabItem>
+</Tabs>
+
+## Prerequisites
+
+### SNMP Configuration
+
+To use this Pack, the SNMP service must be properly configured on your device.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md:95]:
```suggestion
To use this connector, the SNMP service must be properly configured on your device.
```

File: pp/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md [hunk 11]
```diff
@@ -0,0 +1,452 @@
...
+| Macro          | Description                                                                                         | Default value     | Mandatory   |
+|:---------------|:----------------------------------------------------------------------------------------------------|:------------------|:------------|
+| WARNINGUPTIME  | Warning threshold                                                                                   |                   |             |
+| CRITICALUPTIME | Critical threshold                                                                                  |                   |             |
+| EXTRAOPTIONS   | Any extra option you may want to add to the command (E.g. a --verbose flag). All options are listed [here](#available-options) |                   |             |
+
+</TabItem>
+</Tabs>
+
+3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of service, and on page **Resources Status**. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md:259]:
```suggestion
3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.
```

**Review by cg-tw — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md [hunk 12]
```diff
@@ -0,0 +1,454 @@
...
+
+```bash
+dnf install centreon-plugin-Hardware-Sensors-Rittal-Cmc3-Snmp
+```
+
+</TabItem>
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
+
+```bash
+dnf install centreon-plugin-Hardware-Sensors-Rittal-Cmc3-Snmp
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md:170]:
```suggestion
dnf install centreon-plugin-Hardware-Sensors-Rittal-Cmc-Snmp
```

**Review by omercier — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md [hunk 13]
```diff
@@ -0,0 +1,452 @@
...
+| FILTERNAME     | Filter device name (can be a regexp)                                                                                                                                            |                                                     |             |
+| FILTERALIAS    | Filter devices alias (can be a regexp)                                                                                                                                          |                                                     |             |
+| WARNINGSTATUS  | Define the conditions to match for the status to be WARNING (default: '%{status} =~ /detect\|changed\|fwUpdate\|fwUpdateRun/i'). You can use the following variables: %{status} | %{status} =~ /detect|changed|fwUpdate|fwUpdateRun/i |             |
+| CRITICALSTATUS | Define the conditions to match for the status to be CRITICAL (default: '%{status} =~ /notAvail\|lost\|error/i'). You can use the following variables: %{status}                 | %{status} =~ /notAvail|lost|error/i                 |             |
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md:221]:
```suggestion
| WARNINGSTATUS  | Define the conditions to match for the status to be WARNING (default: '%{status} =~ /detect\|changed\|fwUpdate\|fwUpdateRun/i'). You can use the following variables: %{status} | %{status} =~ /detect\|changed\|fwUpdate\|fwUpdateRun/i |             |
| CRITICALSTATUS | Define the conditions to match for the status to be CRITICAL (default: '%{status} =~ /notAvail\|lost\|error/i'). You can use the following variables: %{status}                 | %{status} =~ /notAvail\|lost\|error/i                 |             |
```

File: pp/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md [hunk 14]
```diff
@@ -0,0 +1,452 @@
...
+| UNKNOWNSTATUS  | Define the conditions to match for the status to be UNKNOWN (default: '%{overall\_device\_status} =~ /detected\|lost\|changed\|update/i'). You can use the following variables: %{unit\_status}, %{overall\_device\_status}, %{mode}, %{available\_devices}             | %{overall_device_status} =~ /detected|lost|changed|update/i             |             |
+| WARNINGSTATUS  | Define the conditions to match for the status to be WARNING (default: '%{unit\_status} =~ /overload/i \|\| %{overall\_device\_status} =~ /warning/i'). You can use the following variables: %{unit\_status}, %{overall\_device\_status}, %{mode}, %{available\_devices} | %{unit_status} =~ /overload/i || %{overall_device_status} =~ /warning/i |             |
+| CRITICALSTATUS | Define the conditions to match for the status to be CRITICAL (default: '%{unit\_status} =~ /failed/i \|\| %{overall\_device\_status} =~ /alarm/i'). You can use the following variables: %{unit\_status}, %{overall\_device\_status}, %{mode}, %{available\_devices}    | %{unit_status} =~ /failed/i || %{overall_device_status} =~ /alarm/i     |             |
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/hardware-sensors-rittal-cmc3-snmp.md:244]:
```suggestion
| UNKNOWNSTATUS  | Define the conditions to match for the status to be UNKNOWN (default: '%{overall\_device\_status} =~ /detected\|lost\|changed\|update/i'). You can use the following variables: %{unit\_status}, %{overall\_device\_status}, %{mode}, %{available\_devices}             | %{overall\_device\_status} =~ /detected\|lost\|changed\|update/i             |             |
| WARNINGSTATUS  | Define the conditions to match for the status to be WARNING (default: '%{unit\_status} =~ /overload/i \|\| %{overall\_device\_status} =~ /warning/i'). You can use the following variables: %{unit\_status}, %{overall\_device\_status}, %{mode}, %{available\_devices} | %{unit\_status} =~ /overload/i \|\| %{overall\_device\_status} =~ /warning/i |             |
| CRITICALSTATUS | Define the conditions to match for the status to be CRITICAL (default: '%{unit\_status} =~ /failed/i \|\| %{overall\_device\_status} =~ /alarm/i'). You can use the following variables: %{unit\_status}, %{overall\_device\_status}, %{mode}, %{available\_devices}    | %{unit\_status} =~ /failed/i \|\| %{overall\_device\_status} =~ /alarm/i     |             |
```

**Review by omercier — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**